src/linkManager.ts

```diff
@@ -459,10 +459,11 @@ export class LinkManager {
                     }
                     break;
                 default:
-                    //metadata is not a link, return null
-                    return null;
+                  // We will continue to check other DataView properties
+                  continue
```

Comment:
Just some small nitpick :)

```suggestion
                  continue;
```

Reply:
Oh, we break out from the switch context, that's correct.